com.unity.netcode.gameobjects/Runtime/Transports/UNET/UNetTransport.cs

```diff
@@ -349,28 +349,19 @@ public ConnectionConfig GetConfig()
         {
             var connectionConfig = new ConnectionConfig();
 
-            // Built-in netcode channels
-            for (int i = 0; i < NETCODE_CHANNELS.Length; i++)
+            foreach (int i in Enum.GetValues(typeof(NetworkDelivery)))
             {
-                int channelId = AddNetcodeChannel(NETCODE_CHANNELS[i].Delivery, connectionConfig);
-
-                m_ChannelIdToName.Add(channelId, NETCODE_CHANNELS[i].Channel);
-                m_ChannelNameToId.Add(NETCODE_CHANNELS[i].Channel, channelId);
+//                Console.WriteLine($" {i}" );
             }
 
-            // Custom user-added channels
-            for (int i = 0; i < Channels.Count; i++)
-            {
-                int channelId = AddUNETChannel(Channels[i].Type, connectionConfig);
-
-                if (m_ChannelNameToId.ContainsKey((NetworkChannel)Channels[i].Id))
-                {
-                    throw new InvalidChannelException($"Channel {channelId} already exists");
-                }
-
-                m_ChannelIdToName.Add(channelId, (NetworkChannel)Channels[i].Id);
-                m_ChannelNameToId.Add((NetworkChannel)Channels[i].Id, channelId);
-            }
+            // Built-in netcode channels
```

Comment:
we should probably convert "built-in" channels to use delivery types directly and get rid of this block here in this PR.